Rocket.Chat/Views/Cells/Chat/ChatMessageCell.swift

```diff
@@ -210,7 +210,7 @@ final class ChatMessageCell: UICollectionViewCell {
 extension ChatMessageCell: UIGestureRecognizerDelegate {
 
     func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRequireFailureOf otherGestureRecognizer: UIGestureRecognizer) -> Bool {
-        return false
+        return true
```

Comment:
The event is working fine, but when you return `true` here, the user will need to resign the finger from screen to open the actions. Why did you change it? I think `false` works better.